### PR TITLE
Closes #55

### LINKED ISSUE #55: use hapi 9.x.x as a dependency

### PR DESCRIPTION
**Pull request by lloydbenson**

### CODE REVIEW

File: test/cli.js [hunk 1]
```diff
@@ -287,7 +287,10 @@ describe('Broadcast', function () {
 
                 expect(request.payload.events[0].id).to.equal(TestHelpers.inlineLogEntry.lineTwo.id);
                 reply().code(200);
-                server.stop();
+                server.stop(function () {
```
Comment (arb) [inline test/cli.js]: `Hoek.ignore`

File: test/broadcast.js [hunk 2]
```diff
@@ -207,10 +207,10 @@ describe('Broadcast', function () {
 
         it('logs an error if there is a problem with Wreck', function (done) {
 
-            var log = console.error;
+            var log = console.log;
             var info = console.info;
 
-            console.error = function (value) {
+            console.log = function (value) {
```
Comment (arb) [inline test/broadcast.js]: Did hapi change this somewhere? It's unusual for an error to get logged via `console.log` instead of `console.error`.

Comment (lloydbenson) [inline test/broadcast.js]: Not sure on the error vs console.log but I saw that statusCode specifically that the test was checking was not on error.  I suspect this has to do with a wreck upgrade but I haven't been able to find a specific cause yet.

File: .travis.yml [hunk 3]
```diff
@@ -2,6 +2,4 @@ language: node_js
 
 node_js:
   - 0.10
-  - 0.12
-  - iojs
-
+  - 4
```
Comment (arb) [inline .travis.yml]: `sudo: false`

**Pull request merged**